public/styles.css

```diff
@@ -32,7 +36,7 @@ nav::before {
 }
 
 @media only screen and (min-width: 445px) {
-    nav::before {
+    .logo-small {
         width: 444px;
     }
```

Comment:
This is governing the small logo, so it made sense to refactor out of too many coupled styles.